htdocs/xoops_lib/modules/protector/filters_disabled/postcommon_register_insert_js_check.php

```diff
@@ -60,7 +60,20 @@ public function getHtml4Assign()
         foreach ($as_md5array as $key => $val) {
             $as_md5shuffle[] = array('key' => $key, 'val' => $val);
         }
-        shuffle($as_md5shuffle);
+
```

Comment:
This is just overkill. This isn't a cryptology level use case. This is a high risk change, especially for a feature that is disabled by default.